docs/application/native/tutorials/process/performance.md

```diff
@@ -189,47 +189,138 @@ Select from 2 types of memory profiling settings:
-   | **limit errors reported**       | When enabled, Valgrind stops reporting errors after 10,000,000 in total, or 1,000 different ones, have been seen. This is to stop the error tracking machinery from becoming a huge performance overhead in applications with many errors. |
-   | **show errors below main**      | By default, stack traces for errors do not show any methods that appear beneath `main()`. Alternatively, if `main()` is not present in the stack trace, it does not show any methods below `main()`-like methods, such as glibc's `__libc_start_main()`. Furthermore, if `main()`-like methods are present in the trace, they are normalized (below `main()`), in order to make the output more deterministic. If this option is enabled, all stack trace entries are shown and `main()`-like methods are not normalized. |
-   | **max size of stack frame**     | This option specifies the maximum size of a stack frame. If the stack pointer moves by more than this amount, Valgrind assumes that the application is switching to a different stack.You can to use this option if your application has large stack-allocated arrays. Valgrind keeps track of your application's stack pointer. If it changes by more than the threshold amount, Valgrind assumes your application is switching to a different stack, and **Memcheck** behaves differently than it would for a stack pointer change smaller than the threshold. Usually this heuristic works well. However, if your application allocates large structures on the stack, this heuristic is fooled, and **Memcheck** subsequently reports large numbers of invalid stack accesses. This option allows you to change the threshold to a different value.You must only consider the use of this option if Valgrind's debug output directs you to do so. In that case, it tells you the new threshold you must specify.In general, allocating large structures on the stack is a bad idea, because you can easily run out of stack space, especially on systems with limited memory or which expect to support large numbers of threads each with a small stack, and also because the error checking performed by **Memcheck** is more effective for heap-allocated data than for stack-allocated data. If you have to use this option, consider rewriting your code to allocate on the heap rather than on the stack. |
-   | **suppressions file**           | This option specifies an extra file from which to read descriptions of errors to suppress. You can use up to 100 extra suppression files. |
+
+   - **trace children on exec**
+
+     When enabled, Valgrind traces into sub-processes initiated through the exec system call. This is necessary for multi-project applications. Valgrind does trace into the child of a fork (it would be difficult not to, since fork makes an identical copy of a process), so this option is arguably badly named. However, most children of fork calls immediately call exec anyway.
+
+
+   - **run __libc_freeres()on exit**
```

Comment:
line 198: Add a space. (run __libc_freeres()on exit)
line 238: The separated lines are merged as one line.
line 261: Add a space. (so --show-allwould be)
line 314: The separated lines are merged as one line.
line 356: The separated lines are merged as one line.

Reply:
done